File: trail1/코로나 메뉴얼/covid-manual.cpp
#include <iostream>
using namespace std;

int main() {
    int cnt = 0;
    for (int i=0; i<3; i++) {
        char flu;
        int T;
        cin >> flu >> T;
        if (flu == 'Y' && T >= 37) {cnt++;}
    }
    if (cnt >= 2) {cout << "E" << endl;}
    else {cout << "N" << endl;}
    return 0;
}

File: trail1/코로나 메뉴얼/covid-manual.java
import java.util.Scanner;
public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int cnt = 0;
        for (int i=0; i<3; i++) {
            char flu = sc.next().charAt(0);
            int T = sc.nextInt();
            if (flu == 'Y' && T >= 37) {cnt += 1;}
        }
        System.out.println(cnt >= 2 ? "E" : "N");
    }
}